xnat_tools/__init__.py

```diff
@@ -1 +1 @@
-__version__ = "0.1.3"
+__version__ = "0.1.7"
```

Comment:
Looks like we're duplicating this value in the `pyproject.toml` file. Can we load the `pyproject.toml` version here so there's one source of truth?

Reply:
Or maybe this is automatically updated?

Reply:
Commitizen will update them in both places. 

There are ways to load the version from the `pyproject.toml` but then it breaks other things (which I forget which one... maybe Commitizen it self)

Reply:
That's rad!